es-ds-components/components/es-cta-banner.vue

```diff
@@ -0,0 +1,105 @@
+<script setup>
+const props = defineProps({
+    dark: {
+        type: Boolean,
+        default: false,
+    },
+    hasButton: {
+        type: Boolean,
+        default: false,
+    },
+    // default or wide
+    variant: {
+        type: String,
+        default: 'default',
+    },
+});
+
+const lgFirst = computed(() => {
+    if (props.variant === 'wide') {
+        return props.hasButton ? '8' : '6';
+    }
+
+    return props.hasButton ? '7' : '4';
+});
+
+const lgSecond = computed(() => {
+    if (props.variant === 'wide') {
+        return props.hasButton ? '4' : '6';
+    }
+
+    return props.hasButton ? '5' : '8';
+});
+
+const xxlFirst = computed(() => {
+    if (props.variant === 'wide') {
+        return props.hasButton ? '8' : '6';
+    }
+
+    return props.hasButton ? '8' : '5';
+});
+
+const xxlSecond = computed(() => {
+    if (props.variant === 'wide') {
+        return props.hasButton ? '4' : '6';
+    }
+
+    return props.hasButton ? '4' : '7';
+});
+</script>
+
+<template>
+    <es-card
+        :class="[
+            {
+                'bg-dark-blue border-dark-blue text-white': dark,
+                'px-100 px-lg-200': variant === 'default',
+                'px-100 px-lg-300': variant === 'wide',
+            },
+        ]"
+        v-bind="$attrs">
+        <es-row>
+            <es-col
+                class="mb-200 my-lg-auto text-center text-lg-left"
+                :lg="lgFirst"
+                :xxl="xxlFirst">
+                <!-- avoiding use of an <h2> tag here for long-form content SEO reasons,
+                    but preserving heading semantics for screen readers -->
+                <div
+                    role="heading"
+                    aria-level="2"
+                    class="font-weight-semibold"
+                    :class="{
+                        'font-size-300': variant === 'default',
+                        'mb-50': $slots.subtitle,
+                        'mb-0': !$slots.subtitle,
+                        'text-white': dark,
+                        h2: variant === 'wide',
+                    }">
+                    <slot name="heading"> Easily find what solar costs in your area </slot>
+                </div>
+                <p
+                    v-if="$slots.subtitle"
+                    class="mb-0">
+                    <slot name="subtitle" />
+                </p>
+            </es-col>
+            <es-col
+                class="d-flex justify-content-center justify-content-lg-end my-auto"
+                :lg="lgSecond"
+                :xxl="xxlSecond">
+                <slot name="cta" />
+            </es-col>
+        </es-row>
+    </es-card>
+</template>
+
+<style lang="scss" scoped>
+.es-cta-card {
+    &-subtitle {
+        :deep(p:last-child) {
+            margin-bottom: 0;
```

Comment:
Was this style block copied over from EsCtaCard? I don't think we need it here because (a) we're not applying the class `es-cta-card-subtitle` to anything in this component, and (b) line 83 above puts an `mb-0` on the `<p>` tag surrounding the subtitle.

Reply:
Oh yeah, thanks. At one point I was using your recent `EsCtaCard` PR as a reference and I meant to remove that style block.